README.md

```diff
@@ -277,7 +278,7 @@ setup(React.createElement, undefined, undefined, (props) => {
 });
 ```
 
-The functionality of "transient props" (with a "$" prefix) can be implemented as follows:
+The functionality of "transient props" (with a "\$" prefix) can be implemented as follows:
```

Comment:
It looks like this got hit with the formatter. The rendered result is unchanged👍

Reply:
It's totally fine 😃